frontend/src/App.tsx

```diff
@@ -33,9 +36,28 @@ const wagmiClient = createClient({
     provider,
 });
 
-const maintenance = false;
 
 export const App = () => {
+    let [maintenance, setMaintenance] = useState(false);
```

Comment:
This could probably just be one useState, i.e a state called status with three different values `'up' | 'maintenance' | 'unexpectedDown'`

Fancy typescript for this would probably be `useState<'up' | 'maintenance' | 'unexpectedDown'>('up')`

Reply:
Yeah, just couldn't figure it out the Typescript way (tried an interface, couldn't get that to work. Will implement.

Reply:
Thanks, implemented.